old/industry.csv

```diff
@@ -165,5 +165,5 @@ Arti Ramesh,Binghamton University,https://www.binghamton.edu/cs/people/aramesh.h
 Jo Vermeulen,Aarhus University,http://www.jovermeulen.com,mLqdIdoAAAAJ,TBA
 Ravi Arun Netravali,Princeton University,https://www.cs.princeton.edu/~ravian,4C9naMgAAAAJ,Reulay
 Ravi Netravali,Princeton University,https://www.cs.princeton.edu/~ravian,4C9naMgAAAAJ,Reulay
-Roland Memisevic,University of Montreal,http://www.iro.umontreal.ca/~memisevr,8oJl5P0AAAAJ,Qualcomm
-
+Ivor W. Tsang,University of Technology Sydney,https://www.uts.edu.au/staff/ivor.tsang,rJMOlVsAAAAJ,A*STAR
```

Comment:
Still in `csrankings-i.csv`.

Reply:
have updated. see https://github.com/emeryberger/CSrankings/pull/4959